sample/src/main/java/com/google/android/material/composethemeadapter/sample/BasicIntegration.kt

```diff
@@ -67,7 +68,8 @@ fun ComponentsSample() {
             )
         }
     ) {
-        ScrollableColumn {
+        val state = rememberScrollState()
+        Column(modifier = Modifier.verticalScroll(state)) {
```

Comment:
The double `Column` seems unnecessary. I'd switch this to `LazyColumn` as that's our recommendation.

Reply:
The deprecation message on `ScrollableColumn` says *"... Or you can use Column(Modifier.verticalScroll(rememberScrollState()) if your scrolling content is small enough."* Maybe that's fine given this is a demo?

Reply:
A scrolling Column is fine here imo, but +1 to removing the nested Columns if we can.